src/main/java/com/yanna/stepanova/controller/BookController.java

```diff
@@ -0,0 +1,41 @@
+package com.yanna.stepanova.controller;
+
+import com.yanna.stepanova.dto.BookDto;
+import com.yanna.stepanova.dto.CreateBookRequestDto;
+import com.yanna.stepanova.service.BookService;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping(value = "/books")
```

Comment:
GET: /api/books (Retrieve book catalog)
or put `api/` in application.properties servlet context path property
```suggestion
@RequestMapping("/api/books")
```

Reply:
Can't understand what I should do exactly
There're my screens from Postman:
https://drive.google.com/file/d/1Z5y5h22_gleTP3DLAza4hv_DbhIVT1fw/view?usp=sharing
https://drive.google.com/file/d/1iChV0GI7wAfTxBRCUipPcRJJeld9D3x6/view?usp=sharing